utils/test-utils/src/main/scala/pl/touk/nussknacker/test/BasePatientScalaFutures.scala

```diff
@@ -0,0 +1,26 @@
+package pl.touk.nussknacker.test
+
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.exceptions.TestFailedException
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+trait BasePatientScalaFutures extends ScalaFutures with Eventually {
+
+  implicit class FutureOps[T](future: Future[T]) {
+    // Sometimes we know how long should take time after which some Future will be completed. In this case
```

Comment:
```suggestion
    // Sometimes we know how long we should wait after a Future will be completed. In this case
```

Reply:
I can't agree with this suggestion. We should wait a little bit longer than this assumed time. The time that we should wait is a derivate of time that we assume that future will be completed.

Reply:
Hmmm, but this first sentence is just an introduction to this definition, I think my suggestions has the same meaning (or not?) as it was before.
The wole comment would look like:
```scala
// Sometimes we know how long we should wait until a Future is completed. In this case
// we want to wait a little big longer than this time, to make sure that in case of Failure, inner Failure won't
// be suppressed by exception "A timeout occurred waiting for a future to complete" and extract this inner exception
```
isn't it what you said?

Reply:
I rephrased this comment a bit to make it less complicated.

Reply:
thanks